python-clusters/create-aks-cluster/cluster.json

```diff
@@ -6,6 +6,13 @@
     },
     "architecture": "KUBERNETES",
     "params": [
+        {
+            "name": "s-nodes",
+            "type": "SEPARATOR",
+            "label": "WARNING: Connection parameters overriden by existing legacy settings",
+            "description": "Clear all legacy identity settings in the legacy section to fix it",
```

Comment:
I'd be a very sneaky bastard and inject html here (it's not checked, shush don't tell). For example make the description a <div class="alert alert-warning">...</div> to make it even more visible